japecheck/analyzer.go

```diff
@@ -218,6 +229,23 @@ func parseRouteCall(call *ast.CallExpr, pass *analysis.Pass) *route {
 	case "PUT":
 		r.request = call.Args[1]
 	}
+
+	ast.Inspect(call.Args[0], func(n ast.Node) bool {
+		ident, ok := n.(*ast.Ident)
+		if !ok {
+			return true
+		}
+		t := pass.TypesInfo.Types[ident].Type
+		if t == nil {
+			return false
+		}
+		typString := t.String()
+		if p, ok := t.(*types.Pointer); ok {
+			typString = p.Elem().String()
+		}
+		r.params = append(r.params, typString)
```

Comment:
looks like this can only handle one param. I think we want something like:
- check whether `call.Args[0]` is a call to `fmt.Sprintf`
- if so, loop over its arguments, appending each of them to `r.params`